.github/workflows/test-and-deploy.yml

```diff
@@ -46,6 +46,17 @@ jobs:
           TWILIO_AUTH_TOKEN: ${{ secrets.AUTH_TOKEN }}
         run: make testacc
 
+      - name: Run Test Coverage
+        if: ${{ matrix.go == '1.16' }}
```

Comment:
Shouldn't we also limit this step to PRs, branch pushes & non-fork pushes (line 54)?

Reply:
yup will update it shortly